gradle-plugin/configure/build.gradle.kts

```diff
@@ -1,28 +1,28 @@
-import com.novoda.gradle.release.PublishExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 /// The plugin version number – change this to match whatever your tag will be
-version = "0.5.0"
 group = "com.automattic.android"
+version = "0.6.0"
 
 buildscript {
     repositories {
-        jcenter()
-        maven {
-            setUrl("https://plugins.gradle.org/m2/")
-        }
+        maven { url = uri("https://a8c-libs.s3.amazonaws.com/android") }
     }
     dependencies {
-        classpath("com.novoda", "bintray-release", "0.9.2")
+        classpath("com.automattic.android:publish-to-s3:0.3.1")
     }
 }
 
 plugins {
     `kotlin-dsl`
-    id("com.github.gmazzo.buildconfig") version "2.0.2"
+    id("com.github.gmazzo.buildconfig") version "3.0.0"
 }
 
-apply(null, "com.novoda.bintray-release")
+apply(plugin = "com.automattic.android.publish-plugin-to-s3")
+configure<com.automattic.android.publish.PublishPluginToS3Extension> {
+    groupId = "com.automattic"
```

Comment:
Why did you decide to leave out the `android` domain in the group?

Reply:
That was a mistake 🤦 Fixed in 09f80bc9f46c31fe3b285941521464cd098b046a.